### DISCUSSION TITLE
Auto-play

### DISCUSSION THREAD

**Discussion by JA16122000** [mrlt8/docker-wyze-bridge#574] [Ideas]
Maybe is it an idea, to autoplay when visit your docker URL

**Comment by mrlt8** [comment]
I'll look into this, but FYI, the HLS streams will always have a slight delay compared to the RTSP stream.

**Comment by mrlt8** [comment]
I pushed some changes to the [dev branch](https://github.com/mrlt8/docker-wyze-bridge/pull/577) for auto-play. It works fine in chrome, but Firefox and safari don't seem to respond right now.

**Reply by JA16122000** [reply]
Nice work (!
It seams sort of working (on chrome), but when i try http://localhost:5000/?autoplay it first go to "Snapshot" and not "Video"

**Reply by mrlt8** [reply]
hmm, good point. Technically, the autoplay query just sets the autoplay cookie to true, so you can go back and forth between snapshot and video and it should remember your autoplay option. Will have to think about this some more.

In the meantime, you can also use `/?video&autoplay` or `/?autoplay&video`, and once the cookie set, you should be able to visit the page without the query and it should remember your selection.

**Reply by JA16122000** [reply]
That is working fine indeed!
j---------.nl:5000/?video&autoplay

**Reply by JA16122000** [reply]
Also ://j-*****.nl:5000/?video&autoplay&fullscreen is working fine 😄
